### ISSUE TITLE
time interval between each click request

### ISSUE THREAD

**Issue by location900** [Masuzu/ZooeyBot#11]
I read about time interval between each click request from my Zooeybot.ini file. I find that 'WaitTimeInMsBeforeClickInput=50' is a fixed variable. Will this create an issue that time interval between each request is constant so the server can easily detect that the user is using a bot? 

Another issue is that is zooey bot able to make floating clicks at different positions? I am a bit worried about the bot detection.

**Comment by Masuzu** [comment]
This parameter is not related at all to the delay between clicks. It's something internal between a "mouse down" and "mouse up" event, which is not supposed to be listened to by the webpage (only mouse down events are listened to). I can add a random factor to it.
```
Another issue is that is zooey bot able to make floating clicks at different positions? I am a bit worried about the bot detection.
```
I can do this as for the poker bot. Good point indeed.

**Comment by Masuzu** [comment]
Addressed in version 1.1 https://poker-bot.azurewebsites.net/ZooeyBot/en/Home/PatchNotes
